frontend/packages/dev-console/src/components/guided-tour/GuidedTourText.tsx

```diff
@@ -1,14 +1,14 @@
 import * as React from 'react';
 import { Trans, useTranslation } from 'react-i18next';
 import { PlusCircleIcon } from '@patternfly/react-icons';
-import { useOpenshiftVersion } from '@console/shared/src/hooks/version';
+import { useOpenShiftVersion } from '@console/shared/src/hooks/version';
 import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
 import { K8sResourceKind, referenceForModel } from '@console/internal/module/k8s';
 import { ConsoleLinkModel } from '@console/internal/models';
 
 const DevPerspectiveTourText: React.FC = () => {
   const { t } = useTranslation();
-  const openshiftVersion = useOpenshiftVersion();
+  const openshiftVersion = useOpenShiftVersion();
```

Comment:
Can / Want you want use semver here as well? I think the string slice here is more problematic as the other one.